src/main/java/hudson/plugins/timestamper/io/DumpTimestamps.java

```diff
@@ -50,7 +51,7 @@ public static void main(String... args) throws IOException {
     if (args.length == 0) {
       throw new IllegalArgumentException("no command-line arguments");
     }
-    File timestamperDir = new File(Joiner.on(' ').join(args));
+    File timestamperDir = new File(Util.join(Arrays.asList(args), " "));
```

Comment:
this looks like somehow someone did not know how to quote arguments :-o

Reply:
Indeed, but this is a preexisting issue that I am making neither better nor worse in this PR.